lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -140,15 +140,15 @@ def check_config(self, client, server_config, host, my_headers={}):
 
         try:
             oidc_client = server_config.get("openid-connect", "client")
```

Comment:
FWIW, these lists were in sorted order.

Reply:
I support keeping the list in sorted order.

On another note, @portante, when commenting on multiple lines, you can drag-select to highlight the range, and doing so has two effects:  first, in the "Conversation" display, your comment will be preceded by the code from the range (instead of the default three or so lines), and, second and more importantly, your comment will be displayed _after_ the range (in this instance, it would have made clear what you meant by "these lists").